Header header
string name
string status
bool complete
bool failed
